src/__tests__/x509/cert.test.ts

```diff
@@ -32,36 +30,29 @@ describe('x509Certificate', () => {
         expect(cert.extKeyUsage?.keyCertSign).toBe(true);
         expect(cert.extKeyUsage?.crlSign).toBe(true);
 
-        expect(cert.extAuthorityKeyID).toBeDefined();
```

Comment:
How come this is now undefined?

Reply:
The new cert I generated for testing is intentionally not using an `authorityKeyID` extension so that I cant test different mechanisms for chaining certs together (comparing the issuer and subject of two certs is another way to locate the parent of a given certificate).